### PR TITLE
Amls 4912 - pre-application business matching accessibility updates

### PR DESCRIPTION
**Pull request by samzaroughmrc**
pre-application business matching accessibility updates

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/businessmatching/psr_number.scala.html [hunk 1]
```diff
@@ -41,20 +44,22 @@
...
     heading = header
 )  {
 
     @form(f, controllers.businessmatching.routes.PSRNumberController.post(edit)) {
-        <p>
-            @m("businessmatching.psr.number.hint")
-        </p>
-        @fieldset(
+        @fieldsetWithErrorGroup(
             f = f("appliedfor-fieldset"),
```
Comment (drewmiley) [inline app/views/businessmatching/psr_number.scala.html]: Remove -fieldset

File: app/views/businessmatching/psr_number.scala.html [hunk 2]
```diff
@@ -41,20 +44,22 @@
         labelText = "businessmatching.psr.number.lbl"
     )
 }
-
+@supportingContent = {
+    <p>
+    @m("businessmatching.psr.number.hint")
```
Comment (drewmiley) [inline app/views/businessmatching/psr_number.scala.html]: Move onto one line

File: app/views/businessmatching/services.scala.html [hunk 3]
```diff
@@ -23,7 +23,13 @@
 @header = {
     @components.back_link()
     @errorSummary(f)
-    @heading("msb.services.title", if(isPreSubmission) "summary.businessmatching" else "summary.updateinformation")
+}
+
+@section = @{
+    if(isPreSubmission)
```
Comment (drewmiley) [inline app/views/businessmatching/services.scala.html]: One line , or indent

**Comment by owen-opencast** [comment]
**Testing - Which services do you need to register?**

If reading all content (Control-Option-A on Mac), the VO gets stuck in a loop when it reaches the "What these services mean" drop down.

This looks to be an issue with the detailsSummary template - the <summary> tag needs `role="button"` adding, as per [here](https://design-system.service.gov.uk/components/details/default/index.html). 

This will affect all pages that use the detailsSummary template.

**Comment by drewmiley** [comment]
@owen-opencast Nice spot. Fixed this and pushed it.

**Comment by owen-opencast** [comment]
Passed QA, ready to merge

**Pull request merged**